cosmos/static/cosmos/css/about.css

```diff
@@ -1,25 +1,27 @@
 /* TODO: make these changes more specific so as to change any other carousels unnecesarily */
 
-.testimonials .carousel-inner .carousel-item.active,
-.testimonials .carousel-inner .carousel-item-next,
-.testimonials .carousel-inner .carousel-item-prev {
+.carousel-inner .carousel-item.active,
+.carousel-inner .carousel-item-next,
+.carousel-inner .carousel-item-prev {
```

Comment:
I feel like removing the specificity is only going to lead to more problems later on, cause if we ever want a carousel that is not 3 items wide for example, that is not going to be possible now.